### DISCUSSION TITLE
v7.8.8 causes my app to freeze for several minutes

### DISCUSSION THREAD

**Discussion by NickR49** [react-hook-form/react-hook-form#5647] [General]
With version 7.8.7 my app runs fine, but with 7.8.8 one of the forms freezes on initialisation for several minutes.
I don't have a CSB for this but thought I'd mention it in case anyone is experiencing the same . . .

**Comment by bluebill1049** [comment]
can you do a performance analysis on the dev tool? see which function caused this.
